sphinx_js/templates/function.rst

```diff
@@ -1,5 +1,12 @@
 .. js:function:: {{ name }}{{ params }}
 
+   {% if deprecated -%}
+   .. note::
+
+      This function is deprecated.
```

Comment:
> There's a lot of duplication now between the three directives' templates; I'm wondering if it might be helpful to factor them into a shared ReST template somehow.

If you wanted to factor this stuff out, you could just shorten it to, e.g., "Deprecated. Blah blah blah.", without mention of the kind of construct it is.

Reply:
Btw, I like your choice to render it as a note. I think that gives it the right level of emphasis.

Reply:
Actually, I think the best option would be a colon, since that's how jsdoc renders it by default. Then capital and lowercase letters, as well as complete and incomplete clauses, will read as intended.

"Deprecated: blah blah blah".